pkg/l2serviceattachmentparser/l2serviceattachmentparser.go

```diff
@@ -57,6 +63,22 @@ func (sattp *L2SrvAttParser) ParseL2ServiceAttachment(d *render.RenderData) (str
 		sattp.log.Error(err, "Error occured while handling cni")
 		return "", err
 	}
+	//TODO change picking one subnet if required
+	if len(sattp.subnetResources) > 0 {
+		sp := parser.NewSubnetParser(sattp.subnetResources[0], sattp.log)
```

Comment:
here this will not work correctly in the future JFYI. We need to decide which items from that list will be picked according to the IPAM cni we want to use. And not before that. We should let IPAM cni decide how many items it needs to pick from that list. We can leave it as it is for now. But this should be fixed in the nad-stitching code part. We need to pass the list to the IPAM cni specific code and let IPAM cni if will pick one item of that list (whereabout case) or two items (static case)

Reply:
Yes will take care of this while pushing nad stitching.